package.json

```diff
@@ -28,7 +28,8 @@
     "svelte": "^3.0.0-beta.5"
   },
   "peerDependencies": {
-    "svelte": ">1.44.0"
+    "svelte": ">1.44.0",
+    "webpack": "^3.0.0 || ^4.0.0"
```

Comment:
We maybe also write `"webpack": ">3.0.0"`?

Reply:
Yes, will have to change that, as in #88 someone seems to have the problem with a webpack 2.7 version